github/settings.json

```diff
@@ -6,5 +6,19 @@
     "allow_squash_merge": true,
     "allow_merge_commit": false,
     "allow_rebase_merge": false
-  }
-}
\ No newline at end of file
+  },
+  "branches": [
+    {
+      "name": "master",
+      "required_status_checks": null,
```

Comment:
I assume that the difference between `null` and `false` is that null will not overwrite prior settings?

Reply:
I chose to go at it a different way. If we already have protection settings for a branch (we check that first with an API call), we won't try to apply any changes, to avoid overwriting settings.